spec/intl.html

```diff
@@ -927,6 +927,38 @@ <h1>Temporal.Calendar.prototype.dateUntil ( _one_, _two_, _options_ )</h1>
           </emu-alg>
         </emu-clause>
 
+        <emu-clause id="sup-temporal.calendar.prototype.era">
+          <h1>Temporal.Calendar.prototype.era ( _dateOrDateTime_ )</h1>
+          <p>
+            The `era` method takes one argument _dateOrDateTime_.
+            The following steps are taken:
+          </p>
+          <emu-alg>
+            1. Let _calendar_ be the *this* value.
+            1. Perform ? RequireInternalSlot(_calendar_, [[InitializedTemporalCalendar]]).
+            1. If _calendar_.[[Identifier]] is *"iso8601"*, then
+              1. Return *undefined*.
+            1. Let _era_ be the result of implementation-defined processing of _dateOrDateTime_ and the value of _calendar_.[[Identifier]].
+            1. Return 𝔽(_era_).
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sup-temporal.calendar.prototype.erayear">
+          <h1>Temporal.Calendar.prototype.eraYear ( _dateOrDateTime_ )</h1>
+          <p>
+            The `eraYear` method takes one argument _dateOrDateTime_.
```

Comment:
```suggestion
            The `eraYear` method takes one argument, _dateOrDateTime_.
```
What kind of argument?